### PR TITLE
SiPixelPhase1 online render plugin th2 update

### PR DESCRIPTION
**Pull request by enochnotsocool**
* Stripped unneeded if statements 
* Added styling for TH2 that are not part of OnlineBlock plots

### CODE REVIEW

File: dqmgui/style/SiPixelMapsRenderPlugin.cc [hunk 1]
```diff
@@ -31,11 +31,12 @@ class SiPixelMapsRenderPlugin : public DQMRenderPlugin
 public:
   virtual bool applies( const VisDQMObject & o, const VisDQMImgInfo & )
     {
-      if (o.name.find( "PixelPhase1/Phase1_MechanicalView" ) != std::string::npos || o.name.find( "PixelPhase1/Tracks" ) != std::string::npos || o.name.find( "PixelPilot/" ) != std::string::npos ){
-	  if (o.object && o.name.find( "Coord" ) != std::string::npos ) return true;
-	  return false;
-	}
-      return false;
+      if ((o.name.find( "PixelPhase1/Phase1_MechanicalView" ) != std::string::npos || o.name.find( "PixelPhase1/Tracks" ) != std::string::npos || o.name.find( "PixelPhase1Timing/" ) != std::string::npos  )
+        && o.object && o.name.find( "Coord" ) != std::string::npos )) {
```
Comment (threus) [inline dqmgui/style/SiPixelMapsRenderPlugin.cc]: the last ")" bracket is extra in this "if" statement. could you please check and compile? Thanks.

Comment (enochnotsocool) [inline dqmgui/style/SiPixelMapsRenderPlugin.cc]: Yes, the the last  ')' is wrong. Updated now.

Comment (vanbesien) [inline dqmgui/style/SiPixelMapsRenderPlugin.cc]: Wait... does this mean you created a PR for code that does not compile? Did you actually test?

**Comment by enochnotsocool** [comment]
The code was tested, the "un-compilable code" error  was made when manually
copying the code from our machine to github.

**Comment by threus** [comment]
@BrunoCoimbra @h4d4 could you please merge this PR?
Thanks,
Tomas for DQM

**Pull request merged**